src/webob/acceptparse.py

```diff
@@ -408,6 +408,24 @@ def generator(value):
                 )
         return generator(value=value)
 
+    def _parse_and_normalize_offers(self, offers):
+        """
+        Throw out any offers that do not match the media type ABNF.
+
+        :return: A list of ABNF-validated offers split into the format:
+                 ``[offer_index, offer_type_subtype, offer_media_params]``.
+
+        """
+        lowercased_offers_parsed = []
+        for index, offer in enumerate(offers):
+            match = self.media_type_compiled_re.match(offer.lower())
+            # The regex here is only used for parsing, and not intended to
+            # validate the offer
```

Comment:
This comment is cryptic and confusing (sorry about that) &mdash; could we replace it with something like `Although the offers do not *need* to be validated, by parsing the offers they are necessarily validated in the process. So we drop the offers that are invalid:`

Reply:
fixed by other wording

Reply:
this is merged, do you want to just make a PR to fix these things?

Reply:
Ah, I'm too slow &mdash; ok, will do!